spark/src/main/scala/org/apache/mahout/sparkbindings/io/MahoutKryoRegistrator.scala

```diff
@@ -35,7 +35,6 @@ class MahoutKryoRegistrator extends KryoRegistrator {
     kryo.addDefaultSerializer(classOf[Vector], new WritableKryoSerializer[Vector, VectorWritable])
     kryo.addDefaultSerializer(classOf[DenseVector], new WritableKryoSerializer[Vector, VectorWritable])
     kryo.addDefaultSerializer(classOf[Matrix], new WritableKryoSerializer[Matrix, MatrixWritable])
-    kryo.register(classOf[com.google.common.collect.HashBiMap[String, Int]], new JavaSerializer())
-
+    kryo.register(classOf[BiDictionary])
```

Comment:
Ok, this works without any kryo.register. I've done it both ways. 

Note that there is no separate serializer class, the class extends Serializable but doesn't implement anything special and so should just use the Seq[(K,V)] for one map. The inverse is calculated after the broadcast deserialization so it is never on the wire.

If the rest looks ok I'll remove this line but still not sure why it is bad. Spark recommends using kryo by registering classes. Doesn't removing this mean using default serialization instead of Kryo?